src/docs/development/tools/devtools/memory.md

```diff
@@ -149,86 +185,139 @@ in the legend.
 
 For more information, see [Dart VM internals][vm].
 
-## Event timeline
+## Memory Android Chart (ADB)
+
+DevTools, for an Android app, collects Android's ADB tool parsing the meminfo section of ADB's App Summary section.  This section is the most interesting from a high-level
```

Comment:
"When connected to an Android app, DevTools collects Android ADB meminfo from an ADB app summary. The meminfo section is the most interesting at a high level. If you were to collect this info from the CLI, this is what it would look like:"

Reply:
Done.